content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -141,12 +141,30 @@ const [findRawData, aggregateRawData, commandRawData] =
 
 </TabbedContent>
 
-Instead of immediately awaiting the result of each operation when it's performed, the operation itself is stored in a variable first which later is submitted to the database via a method called `$transaction`. Prisma Client will ensure that either all three `create`-operations succeed or none of them succeed.
+Instead of immediately awaiting the result of each operation when it's performed, the operation itself is stored in a variable first which later is submitted to the database with a method called `$transaction`. Prisma Client will ensure that either all three `create` operations succeed or none of them succeed.
 
 > **Note**: Operations are executed according to the order they are placed in the transaction. Using a query in a transaction does not influence the order of operations in the query itself.
 >
 > Refer to the 📖 [transactions guide](/guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api) for more examples.
 
+From version 4.4.0, the transaction API has a second parameter. For sequential operations, you can use the following optional configuration option in this parameter:
```

Comment:
This wording here is a bit weird, as we are in the section about "Sequential Prisma Client operations" already, not the general "transaction API".

Suggestion:
```suggestion
From version 4.4.0, the sequential operations transaction API has a second parameter. You can use the following optional configuration option in this parameter:
```